bower.json

```diff
@@ -13,7 +13,7 @@
     "angular-ui-router": "~0.2.18",
     "bootstrap": "~3.3.6",
     "ng-file-upload": "^12.1.0",
-    "ng-img-crop": "ngImgCrop#^0.3.2",
+    "ng-img-crop-full-extended": "ngImgCropFullExtended#~6.0.1",
```

Comment:
Bower does this automatically, I think because the package is named `ng-img-crop-full-extended` but the Github repo is `ngImgCropFullExtended`.

For that reason it won't work if we do:
`"ng-img-crop-full-extended": "~6.0.1",`

But we could do this:
`"ngImgCropFullExtended": "~6.0.1",`